app/controllers/home_controller.rb

```diff
@@ -7,4 +7,8 @@ def show
     authorize :home, :show?
     @user =User.find(params[:id])
   end
+
+  def record_not_found
+
+  end
```

Comment:
This method should be available to all controller. Please see the rails default.

Reply:
Done